File: code/coding_pratice_Interview/reverse_array/reverse_array.py
def reverseArray(arr):
    # Method1
    # ans = arr[::-1]

    # Method2
    # For this reverse you have to print arr becoz it change the default array that means it affect the original array.
    # arr.reverse()

    # Method3
    ans1 = list(reversed(arr))

    print(ans1)
reverseArray([1,2,3])

File: code/coding_pratice_Interview/reverse_array/reverse_array.js
const reverseArray = (arr)=>{
    // const ans = arr.reverse() // with plugin
    
    //without plugin
    const ans1 = [] 
    for(let i = arr.length; i > 0; i--){
        ans1.push(i)
    }

    const ans3 = arr.slice().reverse()
    
    console.log(ans3);
}

// reverseArray([1,2,3])